packages/common/container-definitions/src/deltas.ts

```diff
@@ -61,21 +61,94 @@ export interface IDeltaSender {
     flush(): void;
 }
 
-/** Events emitted by the Delta Manager */
+/**
+ * Events emitted by {@link IDeltaManager}.
+ */
+/* eslint-disable @typescript-eslint/unified-signatures */
 export interface IDeltaManagerEvents extends IEvent {
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `messageBuffer`: TODO
+     */
     (event: "prepareSend", listener: (messageBuffer: any[]) => void);
+
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `message`: TODO
+     */
     (event: "submitOp", listener: (message: IDocumentMessage) => void);
+
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `message`: TODO
+     *
+     * - `processingTime`: TODO
+     */
     (event: "op", listener: (message: ISequencedDocumentMessage, processingTime: number) => void);
+
+    /**
+     * TODO
```

Comment:
Seems unused, maybe this is legacy and should be deprecated (though I don't really remember it)?  Seems like it's trying to do the same thing as "saved" event.